.github/CODEOWNERS

```diff
@@ -80,6 +80,7 @@ sycl/include/sycl/ext/oneapi/experimental/invoke_simd.hpp @intel/dpcpp-esimd-rev
 sycl/include/std/experimental/simd.hpp @intel/dpcpp-esimd-reviewers @rolandschulz
 llvm/lib/SYCLLowerIR/LowerInvokeSimd.cpp @intel/dpcpp-esimd-reviewers
 llvm/include/llvm/SYCLLowerIR/LowerInvokeSimd.h @intel/dpcpp-esimd-reviewers
+invoke_simd/ @intel/dpcpp-esimd-reviewers
```

Comment:
It would be good to have a similar change for our internal project as well.

Reply:
i will do this